src/webots/gui/WbView3D.cpp

```diff
@@ -1225,6 +1225,29 @@ void WbView3D::checkRendererCapabilities() {
     }
 #endif
   }
+#ifndef __APPLE__
+  else if (WbSysInfo::isAmdLowEndGpu(WbWrenOpenGlContext::instance()->functions())) {
+    message += tr("Webots has detected that you are using an old AMD GPU. "
+                  "A recent NVIDIA or AMD graphics adapter is highly recommended to run Webots smoothly. ");
+    disableCameraAntiAliasing = true;
+    disableSMAA = true;
+    disableGTAO = true;
+    reduceTextureQuality = 1;
+  }
+#endif
+
+  // check GPU memory
```

Comment:
I would rule out Intel cards from this test as they have 0 GPU memory, but seems to work flawlessly with any texture quality.